floater/test/test_generators.py

```diff
@@ -113,5 +113,7 @@ def test_to_mitgcm_format(fs, tmpdir):
 	    assert os.path.exists(filename)
             array = np.fromfile(filename, dtype='>f4')
             assert len(array)==(fs.Nx*fs.Ny + 1 )* 9
+            npart = fs.N
```

Comment:
None of our tests of `test_to_mitgcm_format()` involve a land mask in which case we would expect that `N = len(lon) =  fs.Nx*fs.Ny`. So the test should pass.